docs/pages/database-access/guides/mongodb-self-hosted.mdx

```diff
@@ -225,7 +225,7 @@ $ tsh db connect example-mongo
 ```
 
 <Admonition type="note" title="Note">
-  The `mongo` command-line client should be available in PATH in order to be
+  The `mongosh` command-line client should be available in PATH in order to be
   able to connect.
```

Comment:
Use same language as in the Atlas guide?

Reply:
Done: 639022b44cd2ba361a1034e5816f5a7eafd7c5b3.